contracts/staking/stakeManager/StakeManagerStorage.sol

```diff
@@ -55,6 +55,7 @@ contract StakeManagerStorage is ProxyStorage, Lockable, RootChainable {
         uint256 activationEpoch;
         uint256 deactivationEpoch;
         uint256 jailTime;
+        uint256 nonce;
```

Comment:
Please don't add nonce into validator struct, keep separate mapping in staking info itself. Since all the required log functions have `validatorId` in it.

Reply:
We don't want to make our validator struct heavy and this obviously makes more sense if we add it in stakingInfo/logger only.